shepherd_bms/src/compute.cpp

```diff
@@ -4,20 +4,63 @@ ComputeInterface::ComputeInterface()
 {
     pinMode(CURRENT_SENSOR_PIN_H, INPUT);
     pinMode(CURRENT_SENSOR_PIN_L, INPUT);
+    pinMode(FAULT_PIN, OUTPUT);
+    initializeCAN(CANLINE_2, CHARGER_BAUD, &(this->chargerCallback));
+    initializeCAN(CANLINE_1, MC_BAUD, &(this->MCCallback));
```

Comment:
We shouldn't make this called "MCCallback", since it is for the general CAN network and not specifically just for the charger, we also need to verify that this callback method works

Reply:
Basically I wanted to redo the framework I used for handling CAN messages, I figured that since all nodes were receiving the same messages then we already know what possible IDs there are, but updating the submodule each time is annoying, especially if we only modify two nodes to utilize it

Reply:
Also we need to figure out a framework for remotely  configuring CAN messages on the car, but that's a later thing for improvements